test/SILGen/magic_identifier_file_conflicting.swift.gyb

```diff
@@ -1,6 +1,6 @@
 // RUN: %empty-directory(%t)
 // RUN: %gyb -D TEMPDIR=%t %s > %t/magic_identifier_file_conflicting.swift
-// RUN: %{python} -c "import sys; t = open(sys.argv[1], 'rb').read().replace('\r\n', '\n'); open(sys.argv[1], 'wb').write(t)" %t/magic_identifier_file_conflicting.swift
+// RUN: %{python} -c "import io; import sys; t = io.open(sys.argv[1], 'r', encoding='utf-8').read().replace('\r\n', '\n'); io.open(sys.argv[1], 'w', encoding='utf-8', newline='\n').write(t)" %t/magic_identifier_file_conflicting.swift
```

Comment:
I don't think `io` is needed for Python3.

Reply:
The test have to be python 2 and python 3 compatible until everything is flipped over.  The `io` path affords us that.  Otherwise we hit issues with bytes-to-text conversion as the inline python was using the binary mode to deal with `newline` not being available in the `open` signature in some python version.

Reply:
Ah, yes.  The `encoding` parameter is otherwise unsupported in Python2.7.